build.gradle

```diff
@@ -56,16 +56,6 @@ allprojects {
 
   jacoco {
     toolVersion = '0.8.11'
-    if (project.tasks.findByName('integrationTest')) {
-      applyTo integrationTest
-    }
-    if (project.tasks.findByName('acceptanceTest')) {
-      applyTo acceptanceTest
-    }
-  }
-
-  jacocoTestReport {
-    dependsOn test
```

Comment:
We should either set this dependency for both `integrationTest` and `test` or remove it at all. I decided to remove it, because in our GHA we're executing tests + generating report as 2 separate steps